mmv1/third_party/terraform/resources/resource_compute_security_policy.go.erb

```diff
@@ -289,7 +289,7 @@ func resourceComputeSecurityPolicy() *schema.Resource {
 				Description: `The URI of the created resource.`,
 			},
 
-		<% unless version == 'ga' -%>
+		<%- unless version == 'ga' -%>
```

Comment:
This creates extra spaces on the next line if `-` isn't set to suppress the leading tabs. Best I can tell, it still generates the GA code just fine.